dom/colorful-legs_test.js

```diff
@@ -0,0 +1,37 @@
+export const tests = []
+
+tests.push(async ({ eq, page }) => {
+  // Click on the button to change the robot's leg colors
+  const button = await page.$('button#leg-color')
+  await button.click()
+
+  // Get the new colors of both legs
+  const legLeftColor = await page.$eval('#leg-left', (node) => getComputedStyle(node).backgroundColor)
+  const legRightColor = await page.$eval('#leg-right', (node) => getComputedStyle(node).backgroundColor)
+
+  // Check if both legs have been assigned the same new color
+  eq(legLeftColor, legRightColor)
+
+  // Ensure the new color is not black
+  eq(legLeftColor !== 'rgb(0, 0, 0)', true, 'The color of the legs should not be black')
```

Comment:
There is nothing in the subject that mentions that black is not a valid new color

Reply:
right, i deleted the test
black isn't a color so the generative IA helping me building the test for this exercice appears coherent to me and i kept this before your input